Exception from real-time package on boot

1. boot atom

**Atom**: 1.21.0-beta2 x64
**Electron**: 1.6.9
**OS**: Mac OS X 10.13
**Thrown From**: [real-time](https://github.com/atom/real-time) package 0.1.0


### Stack Trace

Failed to activate the real-time package

```
At Cannot find module 'etch'

Error: Cannot find module 'etch'
    at Module._resolveFilename (module.js:470:15)
    at Module._resolveFilename (/Applications/Atom.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.get_Module._resolveFilename (/Applications/Atom.app/Contents/Resources/app/src/module-cache.js:354:58)
    at Module.require (/app.asar/static/index.js:40:43)
    at require (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:66:33)
    at /packages/real-time/lib/login-dialog.js:2:14)
    at /packages/real-time/lib/login-dialog.js:88:3)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/src/compile-cache.js:239:29)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:66:33)
    at /packages/real-time/lib/github-auth-token-provider.js:1:176)
    at /packages/real-time/lib/github-auth-token-provider.js:61:3)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/src/compile-cache.js:239:29)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (/app.asar/static/index.js:47:45)
    at require (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:66:33)
    at /packages/real-time/lib/real-time-package.js:12:33)
    at /packages/real-time/lib/real-time-package.js:212:3)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/src/native-compile-cache.js:106:36)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/src/compile-cache.js:239:29)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
```

### Commands

```
     -0:46.5.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-when-scrolling)
```

### Non-Core Packages

```
busy-signal 1.4.3 
file-icons 2.1.12 
go-debug 1.5.2 
go-plus 5.5.5 
go-signature-statusbar 1.2.3 
hyperclick 0.1.5 
intentions 1.1.5 
language-graphql 0.9.0 
language-rust 0.4.12 
linter 2.2.0 
linter-rust 0.8.0 
linter-ui-default 1.6.10 
real-time 0.1.0 
rustfmt 0.1.3 
rustsym 0.4.0 
```

--- a/package.json
+++ b/package.json
@@ -14,11 +14,11 @@
     "@atom/real-time-server": "https://user-with-just-readonly-access-to-realtime-repos:924792417376b236ca11cc234bcc95d306dcd1cb@api.github.com/repos/atom/real-time-server/tarball/v0.13.0",
     "atom-mocha-test-runner": "^1.0.1",
     "deep-equal": "^1.0.1",
-    "etch": "^0.12.6",
     "temp": "^0.8.3"
   },
   "dependencies": {
     "@atom/real-time-client": "https://user-with-just-readonly-access-to-realtime-repos:924792417376b236ca11cc234bcc95d306dcd1cb@api.github.com/repos/atom/real-time-client/tarball/v0.18.2",
+    "etch": "^0.12.6",
     "keytar": "^4.0.4"
   },
   "consumedServices": {

